content/22.vaccines-trad.md

```diff
@@ -290,81 +293,79 @@ Table: Subunit vaccines approved for use in at least one country [@url:https://c
-Additionally, it can be stored at 2 to 8&#176;C.
+Additionally, they can be stored at 2 to 8&#176;C.
 However, the worldwide footprint of Covifenz, and of VLP-based technologies against SARS-CoV-2 broadly, remains small, with only {{owid_VLP_count}} VLP vaccine approved for distribution in {{owid_VLP_countries}} countries (Figure @fig:vlp-distrib).<!--Grammar needs update if anyone outside of Canada approves or if another VLP becomes active--><!--Check automation-->
 Approval and administration of Covifenz in countries outside of Canada has been limited by concerns at the WHO about ties between Medicago and the tobacco industry [@doi:10.1503/cmaj.1095992; @doi:10.1136/bmj.o811].
-While other species of plants have been explored as the upstream bioreactors for plant-derived VLPs, the use of the specific species of tobacco used here increased yield dramatically [@doi:10.4161/hv.22218].
-Therefore, it may be feasible to identify other species of plants that can be used for future vaccines, but the selection of _N. benthamiana_ was not arbitrary.
+While other species of plants have been explored as the upstream bioreactors for plant-derived VLPs, the specific species of tobacco used  increased yield dramatically [@doi:10.4161/hv.22218].
 
 ![
 **Worldwide availability of vaccines developed with VLPs.**
 This figure reflects the number of vaccines based on VLP technology that were available in each country as of {{owid_most_recent_date}}.
-These data are retrieved from Our World in Data [@url:https://ourworldindata.org/coronavirus] and plotted using geopandas [@url:https://github.com/Geopandas/Geopandas].
+These data are retrieved from Our World in Data [@url:https://ourworldindata.org/coronavirus] and plotted using geopandas [@doi:10.5281/zenodo.5573592].
 See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.
 ]({{owid_VLP_map}} "Availability of protein subunit vaccines"){#fig:vlp-distrib secno=1}
 
-##### Trial Safety and Efficacy
+<!-- This figure seems out of place and inconsistent with the text in the Application to COVID-19 section. It should be located near that high-level summary of VLP vaccines, not in the Covifenz section. The text summary also says there is not data available for worldwide distribution, but the map shows the vaccine (Covifenz) in Canada. -->
+
+#### Trial Safety and Efficacy
+<!-- Why do we have a separate section on this and on each of the vaccines? This section made sense for the platforms when we didn't have sections on individual vaccines. Should we merge the Nuvaxovid and Covifenz sections into this one? Merge parts of this section into those sections? Keep it as is? I expected to find this information in the vaccine-specific sections above. -->
```

Comment:
I can work on integrating this text back into the general discussion of the vaccine development. I think the idea was that it would be helpful to see the efficacy data all in one place to get a sense of the general success of the approach, but I do understand why it reads weirdly.

Reply:
After reading the novel vaccines manuscript, I understood the organization here better. But it didn't seem to match the earlier platforms in this manuscript very well.